File: app/src/main/java/com/lrs/lrsmeeting/base/interfaceOnclick/OnAdapterOnclick.java
package com.lrs.lrsmeeting.base.interfaceOnclick;


import com.lrs.lrsmeeting.base.bean.BaseBean;

/**
 * @description 作用: adapter点击事件
 * @date: 2019/11/4
 * @author: 卢融霜
 */
public interface OnAdapterOnclick {
    /**
     * adapter 点击事件
     *
     * @param position 索引
     * @param bean     实体数据
     */
    void onClickListener(int position, BaseBean bean);
}

File: app/src/main/java/com/lrs/lrsmeeting/activity/meeting/onInterface/OnAdapterOnclick.kt
package com.lrs.lrsmeeting.activity.meeting.onInterface

/**
 * @description 作用:
 * @date: 2020/4/9
 * @author: 卢融霜
 */
interface OnAdapterOnclick {
    /**
     * adapter 点击事件
     *
     * @param position 索引
     */
    fun onClickListener(position: Int, state: Boolean)
}